app/stac_api/serializers.py

```diff
@@ -85,6 +89,75 @@ def data(self):
         return ReturnDict(ret, serializer=self)
 
 
+class LandingPageLinkSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = LandingPageLink
+        fields = ['href', 'rel', 'link_type', 'title']
+
+
+class LandingPageSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = LandingPage
+        fields = ['id', 'title', 'description', 'links', 'stac_version']
+
+    id = serializers.CharField(max_length=255, source="name")
+    links = LandingPageLinkSerializer(many=True, read_only=True)
+    stac_version = serializers.SerializerMethodField()
+
+    def get_stac_version(self, obj):
+        return STAC_VERSION
+
+    def to_representation(self, instance):
+        representation = super().to_representation(instance)
+        api_base = settings.API_BASE
+        request = self.context.get("request")
+        # Add auto links
+        # We use OrderedDict, although it is not necessary, because the default serializer/model for
+        # links already uses OrderedDict, this way we keep consistency between auto link and user
+        # link
+        representation['links'][:0] = [
+            OrderedDict([
+                ('rel', 'self'),
+                ('href', request.build_absolute_uri(f'/{api_base}')),
+                ("type", "application/json"),
+                ("title", "This document"),
+            ]),
+            OrderedDict([
+                ("rel", "service-doc"),
+                ("href", request.build_absolute_uri(f"/{api_base}api.html")),
+                ("type", "text/html"),
+                ("title", "The API documentation"),
+            ]),
+            OrderedDict([
+                ("rel", "service-desc"),
+                ("href", request.build_absolute_uri(f"/{api_base}openapi.yaml")),
```

Comment:
this is an error in the current landing page, should be `/{api_base}/static/openapi.yaml` and `/{api_base}/static/api.html`

Reply:
Ok I'll change it and also in the spec.

Reply:
Done